src/Styles/AddBook.css

```diff
@@ -0,0 +1,51 @@
+.add-form {
+  margin-top: 20px;
+  display: flex;
+  gap: 32px;
+}
```

Comment:
- [ ] There is a grey line that divides the book list from the form, please 🙏🏽 add it to be similar to the original design and make sure to use the same background, in the original design the add form background is grey similar to the background of the entire page.

![image](https://github.com/Wils-FOURR-speed/bookstore/assets/58642949/42b70792-8f4b-4449-8ede-da61e1abd8a8)